### ISSUE TITLE
Match margin, highlighting color is messed up

### ISSUE THREAD

**Issue by martinskuta** [Microsoft/VS-PPT#206]
Today visual studio updated my Match Margin addin to 15.0.6 version and the highlighting colors are eye burning and unreadable?! I didn't find anything related to this in release notes. This is how it looks like now:

![image](https://user-images.githubusercontent.com/14070352/46953492-4a94cf80-d08e-11e8-837e-e81cb31574bc.png)

It might be related to theme colors, because in Light theme it actually looks ok:

![image](https://user-images.githubusercontent.com/14070352/46953426-25a05c80-d08e-11e8-83ba-da163ed2bebc.png)

I am using VS 15.8.7. Is it just me having this white highlighting color in Dark mode?

**Comment by StevenTCramer** [comment]
Yes in dark mode Match Margin isn't good.  But who doesn't use dark mode ;)

**Comment by rkhoury** [comment]
This was reported quite a while ago now, and while I find that my editor highlight colour isn't a problem any more (perhaps I customised it somehow - I can't remember now) I find that with the **Show matches in the editor** option on that cursor-based highlighting in the _Output_ pane has exactly this problem.  Does anyone else experience this?

**Comment by bobinush** [comment]
Same issue in the _Find Results_ window in dark mode, No option to change match color for that window in Tools - Options - Environment - Fonts and Colors
![image](https://user-images.githubusercontent.com/12391559/80005148-7a166a80-84c3-11ea-9598-b3292b0414f9.png)

**Comment by dlackey-aegistg** [comment]
For anyone finding this via a search about Match Margin color issues, I'll describe what I've found.
Open the menu Tools->Options->Environment->Fonts and Colors, then scroll Display Items to **Match color**. _Item background_ is the highlight color in the text editor, and _Item foreground_ is the color of the dot in the scroll bar that points to word matches.

Just note that there does not seem to be a way to set the foreground (text) color of the matches in the text editor, so you should try to choose a color for Item background that contrasts well with the other text colors in your theme.

Also, setting the Item foreground to Automatic will cause the scroll bar dot to be invisible. That was the issue I was having after trying to change the default bright pinky/purpley background color and hoping to set the font color as well.

Hope this helps!

![2020-10-16 10_32_37-Options](https://user-images.githubusercontent.com/72475589/96278510-32b4f680-0f9b-11eb-96c9-f103f2f4adf0.png)

**Comment by hanushasl** [comment]
> Same issue in the _Find Results_ window in dark mode, No option to change match color for that window in Tools - Options - Environment - Fonts and Colors
> ![image](https://user-images.githubusercontent.com/12391559/80005148-7a166a80-84c3-11ea-9598-b3292b0414f9.png)

Regarding the issue with highlighting in Find Results window - I could get rid of it by unchecking the following option.

![image](https://user-images.githubusercontent.com/67586713/125235922-7f87ec80-e2e3-11eb-942a-7ab912178642.png)

Hope this helps!
